sys/mips/mips/trap.c

```diff
@@ -1464,10 +1475,7 @@ MipsEmulateBranch(struct trapframe *framePtr, trapf_pc_t _instPC, int fpcCSR,
 	(InstPtr + 4 + ((short)inst.IType.imm << 2))
 
 	if (instptr) {
-		if (!KERNLAND(instptr))
-			inst.word = fuword32((void *)instptr); /* XXXAR: error check? */
-		else
-			inst = *(InstFmt *) instptr;
+		inst = *(InstFmt *) instptr;
 	} else {
 		if (!KERNLAND((__cheri_addr vaddr_t)instPC))
 			inst.word = fuword32_c(instPC);  /* XXXAR: error check? */
```

Comment:
Not sure if these are in future batches, but should ditch `_c` for these as well.

Reply:
My plan is to rebase my other pending changes to fix remaining callers in the kernel after this branch is pushed and get those merged, then to do a sweep renaming all the _c variants and removing them.